apps/drupal-default/particle_theme/templates/block/block--utc-hero-video--full-reverse.html.twig

```diff
@@ -0,0 +1,152 @@
+{#
+/**
+ * @file Video Hero!
+ * Implements a video hero block in dark blue
+ *
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to the cards. 
```

Comment:
Is this comment relevant? I guess this was copied from cards, but I don't think this does mutiple hero or cards?